tests/sysauth/sssd.pm

```diff
@@ -74,7 +74,7 @@ sub run {
 
     foreach my $scenario (@scenario_list) {
         # Download the source code of test scenario
-        script_run "cd ~/sssd && mkdir $scenario && curl -L -v " . autoinst_url . "/data/sssd-tests/$scenario > $scenario/cdata";
+        script_run "cd ~/sssd && mkdir -p $scenario && curl -L -v " . autoinst_url . "/data/sssd-tests/$scenario > $scenario/cdata";
```

Comment:
The directories are already created when downloading the data: https://github.com/os-autoinst/os-autoinst-distri-opensuse/tree/bfe6ebfe94d9c2eda005afc7e7d3a6ba2e182ba1/data/sssd-tests

I suppose the mkdir command should be removed.

```suggestion
        script_run "cd ~/sssd && curl -L -v " . autoinst_url . "/data/sssd-tests/$scenario > $scenario/cdata";
```

Reply:
okay, checked this and provide new test verification run, thanks!